index.html

```diff
@@ -52,13 +53,12 @@
         </button>
       </div>
     </section>
-    <section class="container mx-12 mt-12 mb-12">
-      <div class="grid gap-8 md:grid-cols-2 lg:grid-cols-4">
+    <section class="  md:mx-12 mx-12  md:mb-12 mb-8">
+      <div class="grid gap-8 grid-cols-1 md:grid-cols-2   lg:grid-cols-4">
         <!-- super power, this is my soft skills-->
         <div
           class="
-            w-72
-            h-60
+           
```

Comment:
It looks like prettier is not working on your vscode... I'd recommend you try to reinstall it